gx_suite_definitions/gene_info.ipynb

```diff
@@ -0,0 +1,433 @@
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# biodomains\n",
+    "validator.expect_column_values_to_be_of_type(\"biodomains\", \"list\")\n",
+    "validator.expect_column_values_to_have_list_members_of_type(column=\"biodomains\", member_type=\"str\")\n",
+    "validator.expect_column_values_to_have_list_members(column=\"biodomains\", list_members={\n",
+    "        'Apoptosis',\n",
+    "        'Vasculature',\n",
+    "        'Lipid Metabolism',\n",
+    "        'Metal Binding and Homeostasis',\n",
+    "        'Proteostasis',\n",
+    "        'Immune Response',\n",
+    "        'Cell Cycle',\n",
+    "        'DNA Repair',\n",
+    "        'Autophagy',\n",
+    "        'Mitochondrial Metabolism',\n",
+    "        'Structural Stabilization',\n",
+    "        'Synapse',\n",
+    "        'Endolysosome',\n",
+    "        'Oxidative Stress',\n",
+    "        'Epigenetic',\n",
+    "        'APP Metabolism',\n",
+    "        'RNA Spliceosome',\n",
+    "        'Tau Homeostasis',\n",
+    "        'Myelination'\n",
+    "    }\n",
+    ")"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# is_adi\n",
+    "validator.expect_column_values_to_be_of_type(\"is_adi\", \"bool\")\n",
+    "validator.expect_column_values_to_not_be_null(\"is_adi\")"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# is_tep\n",
+    "validator.expect_column_values_to_be_of_type(\"is_tep\", \"bool\")\n",
+    "validator.expect_column_values_to_not_be_null(\"is_tep\")"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# resource_url\n",
+    "validator.expect_column_values_to_be_of_type(\"resource_url\", \"str\")\n",
+    "validator.expect_column_values_to_match_regex(\"resource_url\", \"^https://adknowledgeportal.synapse.org/Explore/Target\")"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# ensembl_info\n",
+    "validator.expect_column_values_to_be_of_type(\"ensembl_info\", \"str\")\n",
+    "validator.expect_column_values_to_not_be_null(\"ensembl_info\")\n",
+    "with open(\"../src/agoradatatools/great_expectations/gx/json_schemas/gene_info/ensembl_info.json\", \"r\") as file:\n",
+    "    ensembl_info_schema = json.load(file)\n",
+    "validator.expect_column_values_to_match_json_schema(\"ensembl_info\", ensembl_info_schema)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# multi-field\n",
+    "validator.expect_compound_columns_to_be_unique([\"hgnc_symbol\", \"ensembl_gene_id\"])"
```

Comment:
Is this redundant? We already have an expectation of `ensembl_gene_id` being unique, so by default `ensembl_gene_id` + `hgnc_symbol` will also be unique, right?

Reply:
Yes, I suppose it is! This came from the R [rules](https://github.com/JessterB/agora-data-validation/blob/275dd46f48a69ec23da8a021a793da1036beb0b7/src/rules/rules_gene_info.R#L97).

Reply:
I think we can probably remove this expectation but I'll leave it up to @JessterB

Reply:
removed in latest commit